src/libs/antares/study/parameters.cpp

```diff
@@ -1783,7 +1756,7 @@ void Parameters::saveToINI(IniFile& ini) const
 
     // Seeds
     {
-        auto* section = ini.addSection("seeds - Mersenne Twister");
+        auto* section = ini.addSection("seeds - Mersenne Twister"); //@TODO : Why Capital letters are used here ?
```

Comment:
I suggest you keep this as is for compatibility reasons.

Possible explanation [here](https://fr.wikipedia.org/wiki/Mersenne_Twister).

Reply:
OK. See [commit ](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/342/commits/a5a76f731abb174f417a1e70ec7f1fd996da2981)